Scripts/Extensions/Engine/AudioMotionExtensions.cs

```diff
@@ -0,0 +1,76 @@
+using UnityEngine;
+using FriedSynapse.FlowEnt.Motions.Audio;
+
+namespace FriedSynapse.FlowEnt
+{
+    public static class AudioMotionExtensions
+    {
+        #region Pitch
+
+        /// <summary>
+        /// Lerps the pitch of an <see cref="AudioSource" /> from its current pitch to its current pitch plus the given <paramref name="value" />.
+        /// <para>NOTE: Final value is clamped between the minimum and maximum possible pitch values. i.e -3 to 3 respectively</para>
+        /// </summary>
+        /// <param name="motion"></param>
+        /// <param name="value"></param>
```

Comment:
put the limitations of this parameter info in here rather than in the summary

Reply:
if you put the limitation in the param tags it doesnt show up in intellisense tooltip 😢

Reply:
👍